src/main/java/org/quantumbadger/redreader/activities/MainActivity.java

```diff
@@ -555,8 +631,10 @@ public void onRedditAccountChanged() {
 	@Override
 	protected void doRefresh(final RefreshableFragment which, final boolean force, final Bundle savedInstanceState) {
 
-		if(which == RefreshableFragment.MAIN_RELAYOUT) {
+		if(which == RefreshableFragment.MAIN_RELAYOUT || which == RefreshableFragment.POSTS) {
 
+			actionbar = getSupportActionBar();
+			actionbar.setDisplayHomeAsUpEnabled(false);
```

Comment:
I noticed in the screen capture that this button was now the app icon. Presumably this should either be a back arrow (with the touch area extended to cover the text, as before), or a hamburger icon, depending on the value of the preference?